### PR TITLE
Basic Tests for Q, W, Shift+Q, Shift+W

### PR DESCRIPTION
**Pull request by ericfont**
Tests for basic functionally.  Going between 128th duration & whole duration and applies commands 'half-duration', 'double-duration', 'dec-duration-dotted', 'inc-duration-dotted'

Class Score gets methods cmdDoubleDuration(), cmdHalfDuration(), cmdIncDurationDotted(), cmdDecDurationDotted() which are public wrappers around private method cmdIncDecDurationDotted.

Simplified the text description of the commands

### CODE REVIEW

File: libmscore/cmd.cpp [hunk 1]
```diff
@@ -3172,10 +3172,10 @@ void Score::cmd(const QAction* a)
...
             { "reset-stretch",              [this]{ resetUserStretch();                                         }},
             { "mirror-note",                [this]{ cmdMirrorNoteHead();                                        }},
-            { "double-duration",            [this]{ cmdIncDecDuration(-1, false);                               }},
-            { "half-duration",              [this]{ cmdIncDecDuration(1, false);                                }},
-            { "inc-duration-dotted",        [this]{ cmdIncDecDuration(-1, true);                                }},
-            { "dec-duration-dotted",        [this]{ cmdIncDecDuration(1, true);                                 }},
+            { "double-duration",            [this]{ cmdDoubleDuration();                                        }},
+            { "half-duration",              [this]{ cmdHalfDuration();                                          }},
+            { "inc-duration-dotted",        [this]{ cmdIncDurationDotted();                                     }},
+            { "dec-duration-dotted",        [this]{ cmdDecDurationDotted();                                     }},
```
Comment (ericfont) [inline libmscore/cmd.cpp]: I think these wrapper functions make code easier to read.

File: libmscore/score.h [hunk 2]
```diff
@@ -570,6 +570,10 @@ class Score : public QObject, ScoreElement {
       void cmdAddOttava(OttavaType);
       void cmdAddStretch(qreal);
       void cmdResetNoteAndRestGroupings();
+      void cmdDoubleDuration()      { cmdIncDecDuration(-1, 0); }
+      void cmdHalfDuration()        { cmdIncDecDuration( 1, 0); }
+      void cmdIncDurationDotted()   { cmdIncDecDuration(-1, 1); }
+      void cmdDecDurationDotted()   { cmdIncDecDuration( 1, 1); }
```
Comment (ericfont) [inline libmscore/score.h]: public wrapper functions around the private function cmdIncDecDuration

File: mscore/shortcut.cpp [hunk 3]
```diff
@@ -2499,8 +2499,8 @@ Shortcut Shortcut::_sc[] = {
          MsWidget::SCORE_TAB,
          STATE_NORMAL | STATE_NOTE_ENTRY_STAFF_PITCHED | STATE_NOTE_ENTRY_STAFF_DRUM /*| STATE_NOTE_ENTRY_TAB*/,
          "dec-duration-dotted",
-         QT_TRANSLATE_NOOP("action","Decrease Duration to Next Smaller Dotted or Non-Dotted Duration"),
-         QT_TRANSLATE_NOOP("action","Decrease duration to next smaller dotted or non-dotted duration"),
+         QT_TRANSLATE_NOOP("action","Decrease Duration Dotted"),
+         QT_TRANSLATE_NOOP("action","Decrease duration dotted"),
```
Comment (ericfont) [inline mscore/shortcut.cpp]: The command's description has been simplified, for reasons I mentioned in: https://musescore.org/en/node/175461#comment-682611

**Comment by ericfont** [comment]
note this should go in both master and 2.1, I think.

**Comment by lasconic** [comment]
2.1 is too different to merge this automatically. Would you mind creating a new PR ?

**Comment by ericfont** [comment]
ok, I'll create a new PR.

**Pull request merged**